pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -647,8 +647,35 @@ func TestNetworkGraph(t *testing.T) {
 		},
 		PktType: "OUTGOING",
 		Proto:   "tcp",
-		Addr:    "1.1.1.1",
-		Port:    443,
+		// There's no place like
+		Addr: "127.0.0.1",
+		Port: 80,
+	}
+
+	if !reflect.DeepEqual(event, expectedEvent) {
+		t.Fatalf("Received: %+v, Expected: %+v", event, expectedEvent)
+	}
+
+	// check network-graph HOST event
+	result = <-ch
+	event = networktypes.Event{}
+	if err := json.Unmarshal([]byte(result), &event); err != nil {
+		t.Fatalf("failed to unmarshal json: %s", err)
+	}
+
+	expectedEvent = networktypes.Event{
+		Event: eventtypes.Event{
+			Type: eventtypes.NORMAL,
+			CommonData: eventtypes.CommonData{
+				Node:      "local",
+				Namespace: "default",
+				Pod:       "test-local-gadget-network-graph001",
+			},
+		},
+		PktType: "HOST",
```

Comment:
> Correct, the network-graph does not keep the order. So the test must be modified to accept any order.
>
I also reproduced it. Maybe we could capture all the `eventtypes.NORMAL` events and reuse this to verify the expected entries:

https://github.com/inspektor-gadget/inspektor-gadget/blob/14da590a4d5edf7d825f36cbb20fc1ef0612d184/integration/helpers.go#L107-L118